cmd/pithosctl/status.go

```diff
@@ -177,8 +206,10 @@ func isClusterHealthy(status *cluster.Status) (unhealthyReason string, healthy b
 				return fmt.Sprintf("cannot parse load from cassandra node %s", nodeJ.Address), false
 			}
 
-			if math.Abs((float64(loadI-loadJ))/math.Max(float64(loadI), float64(loadJ))) > 0.2 {
-				return fmt.Sprintf("cassandra load on node (pod %s, %s) is not equal to load on node (pod %s, %s)", nodeI.Address, nodeI.Load, nodeJ.Address, nodeJ.Load), false
+			if math.Min(float64(loadI), float64(loadJ)) > float64(minimumLoadThreshold) {
```

Comment:
Again, this looks like a slightly overloaded function with a lot of detail packed into it.
I'd at least try to provide some insight into computations by abstracting them into functions with descriptive names:
```
if isLoadOverThreshold(load1, load2) {
		return reasonUnequalLoad(node1, node2), false
}
```